frontend/packages/console-shared/src/components/formik-fields/multi-column-field/MultiColumnField.tsx

```diff
@@ -23,9 +24,14 @@ const MultiColumnField: React.FC<MultiColumnFieldProps> = ({
   disableDeleteRow,
   disableAddRow,
   toolTip,
+  spans,
 }) => {
   const { values } = useFormikContext<FormikValues>();
   const fieldValue = _.get(values, name, []);
+  const totalFieldCount: gridItemSpanValueShape = React.Children.count(
+    children,
+  ) as gridItemSpanValueShape;
```

Comment:
Typing and casting should not be needed... did TypeScript yell at you?

Reply:
yes, after changing the type of ```totalFieldCount``` to ```gridItemSpanValueShape``` from ```number``` in the utils.

Reply:
Sorry, i caused this because I didn't follow the code while making the suggestion for the input param type.
Don't care that much which approach you take so long as the `getSpans` function supports a number > 12.